src/utilities/UPENN/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+#
+set(dir utilities/UPENN)
+
+set(dir_EXE_SOURCES ${dir}_EXE_SOURCES)
+
+set(${dir_EXE_SOURCES} ${UPENN_libgeom}
```

Comment:
this seems incorrect. They are dependencies. There are no sources here as far as I can see, so delete this file?

Reply:
There are a few utilities that I have not cleaned up, yet. But I have them on my side.

Reply:
in any case, libgeom etc are not sources

Reply:
How should I add the dependencies then?

Reply:
Temporality, I commented this out

Reply:
> How should I add the dependencies then?

`target_link_libraries`